benchmarks/scripts/benchmarks.py

```diff
@@ -225,52 +259,76 @@ def upload_jar_to_cluster(self, cluster_hostname, ssh_id_file, delta_version_to_
         jar_local_path = out.decode("utf-8").strip()
         jar_remote_path = f"{self.benchmark_id}-benchmarks.jar"
         scp_cmd = \
-            f"scp -C -i {ssh_id_file} {jar_local_path} hadoop@{cluster_hostname}:{jar_remote_path}"
+            f"scp -C -i {ssh_id_file} {jar_local_path} {ssh_user}@{cluster_hostname}:{jar_remote_path}"
         print(scp_cmd)
         run_cmd(scp_cmd, stream_output=True)
         print(">>> Benchmark JAR uploaded to cluster\n")
         return f"~/{jar_remote_path}"
 
-    def start_benchmark_via_ssh(self, cluster_hostname, ssh_id_file, jar_path):
+    def install_dependencies_via_ssh(self, cluster_hostname, ssh_id_file, ssh_user):
+        script_file_name = f"{self.benchmark_id}-install-deps.sh"
+        script_file_text = """
+#!/bin/bash
+packagesNeeded='screen'
```

Comment:
nit: this script will attempt to install screen everytime the benchmark is run. it might be better to check whether screen exists first before attempting to install it.

Reply:
I extended the bash script as suggested.